Code/Mantid/docs/source/algorithms/CreateMD-v1.rst

```diff
@@ -102,4 +105,42 @@ Output
    [0,0,L]
    DeltaE
 
+**Multi Conversion Example InPlace**
+##########################################
+
+.. testcode:: MultiConversionInPlace
+
+   # Create multiple runs 
+   input_runs = list()
+   psi = list()
+   gs = list()
+   gl = list()
+   for i in range(1, 5):
+       current_ws = CreateSimulationWorkspace(Instrument='MAR', BinParams=[-3,1,3], UnitX='DeltaE', OutputWorkspace='input_ws_' + str(i))
+       input_runs.append(current_ws.name())
+       psi.append(float(5 * i))
+       gl.append(0.0)
+       gs.append(0.0)
+    
+   # Convert and merge. ConversionToMD done in place.
+   new_merged = CreateMD(input_runs, Emode='Direct', Alatt=[1.4165, 1.4165,1.4165], Angdeg=[ 90, 90, 90], u=[1, 0, 0,], v=[0,1,0], Psi=psi, Gl=gl, Gs=gs, EFix=3.0, InPlace=True)
+
+   # Show dimensionality and dimension names
+   ndims = new_merged.getNumDims()
+   for i in range(ndims):
+       dim = new_merged.getDimension(i)
+       print dim.getName()
+
+Output
+^^^^^^
+
+.. testoutput:: MultiConversionInPlace
+
+   [H,0,0]
+   [0,K,0]
+   [0,0,L]
+   DeltaE
+
+.. categories::
+
 .. categories::
```

Comment:
I don't think we need both `.. categories::` lines. It produces 2 copies of the same line at the bottom of the page.